src/Classes/MyRadioEmail.php

```diff
@@ -200,11 +187,15 @@ private function getHeader()
 
     private static function addFooter($message)
     {
-        return $message.self::$rtnl.self::$rtnl.self::$footer;
+        $footer = 'This email was sent automatically from MyRadio. '
+            .'You can opt out of emails by visiting https:'.Config::$base_url.'Profile/edit/.';
```

Comment:
Thoughts on changing these to use the urlutils generator?

Reply:
Probably could do, now that they're definitely not static class variables

I'd like to fix it so base_url doesn't expect to have a trailing slash (unlike all the other config domains...), but I'm not sure how much work that would be